Crash report 3.0.2: NPE in WPActionBarActivity

```
java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=100, result=-1, data=null} to activity {org.wordpress.android/org.wordpress.android.ui.posts.PreviewPostActivity}: java.lang.NullPointerException
       at android.app.ActivityThread.deliverResults(ActivityThread.java:3319)
       at android.app.ActivityThread.handleSendResult(ActivityThread.java:3362)
       at android.app.ActivityThread.access$1100(ActivityThread.java:141)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1282)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:5041)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:793)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:560)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by: java.lang.NullPointerException
       at org.wordpress.android.ui.WPActionBarActivity$2.onScroll(WPActionBarActivity.java:337)
       at android.widget.AbsListView.invokeOnItemScrollListener(AbsListView.java:1340)
       at android.widget.AbsListView.setOnScrollListener(AbsListView.java:1329)
       at org.wordpress.android.ui.WPActionBarActivity.initMenuDrawer(WPActionBarActivity.java:329)
       at org.wordpress.android.ui.WPActionBarActivity.initMenuDrawer(WPActionBarActivity.java:276)
       at org.wordpress.android.ui.WPActionBarActivity.onActivityResult(WPActionBarActivity.java:619)
       at android.app.Activity.dispatchActivityResult(Activity.java:5293)
       at android.app.ActivityThread.deliverResults(ActivityThread.java:3315)
       at android.app.ActivityThread.handleSendResult(ActivityThread.java:3362)
       at android.app.ActivityThread.access$1100(ActivityThread.java:141)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1282)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:5041)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:793)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:560)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

--- a/WordPress/src/main/java/org/wordpress/android/ui/WPActionBarActivity.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/WPActionBarActivity.java
@@ -308,17 +308,6 @@ public abstract class WPActionBarActivity extends Activity {
                 // if we have an intent, start the new activity
             }
         });
-        mListView.setOnScrollListener(new AbsListView.OnScrollListener() {
-            @Override
-            public void onScrollStateChanged(AbsListView view, int scrollState) {
-            }
-
-            @Override
-            public void onScroll(AbsListView view, int firstVisibleItem, int visibleItemCount,
-                    int totalItemCount) {
-                mMenuDrawer.invalidate();
-            }
-        });
 
         mMenuDrawer.setMenuView(mListView);
         mListView.setAdapter(mAdapter);

